assets/stylesheets/bootstrap/mixins/_nav-divider.scss

```diff
@@ -1,10 +1,11 @@
+@use "sass:math";
 // Horizontal dividers
 //
 // Dividers (basically an hr) within dropdowns and nav lists
 
 @mixin nav-divider($color: #e5e5e5) {
   height: 1px;
-  margin: (($line-height-computed / 2) - 1) 0;
+  margin: math.div(($line-height-computed, 2) - 1) 0;
```

Comment:
should be `margin: (math.div($line-height-computed, 2) - 1) 0;`

Reply:
fixed